manager/api/tests/test_jira.py

```diff
@@ -311,6 +316,31 @@ def test_needed_parameters(self):
 
         mock_jira_patcher.stop()
 
+    def test_update_time_loss(self):
+        """Test call to update_time_loss and verify field was updated"""
+        # patch both requests.get, requests.put
+        mock_jira_patcher = patch("requests.get")
+        mock_jira_client = mock_jira_patcher.start()
+        response = requests.Response()
+        response.status_code = 200
+        response.json = lambda: {"customfield_10106": 13.6}
+        mock_jira_client.return_value = response
+
+        mock_jira_patcher = patch("requests.put")
+        mock_jira_client = mock_jira_patcher.start()
+        response = requests.Response()
+        response.status_code = 200
+        mock_jira_client.return_value = response
```

Comment:
Replace with patch decorators to simplify. No need to create response twice, me thinks.

@patch("requests.get")
@patch("requests.post")
def test_update_time_loss(self, mock_get, mock_post):
        """Test call to update_time_loss and verify field was updated"""
        # patch both requests.get, requests.put
        response = requests.Response()
        response.status_code = 200
        mock_get.return_value = response

        response.json = lambda: {"customfield_10106": 13.6}
        mock_post.return_value = response

Reply:
I considered that and saw the pattern in this file was to use the decorators for patching environment variables and the implemented way for `requests`. I wasn't familiar with any best practices that designate one way or the other. However I do like this way so I'll update it.